packages/xrpl/src/models/transactions/accountSet.ts

```diff
@@ -155,10 +155,14 @@ const MAX_TICK_SIZE = 15
  * @param tx - An AccountSet Transaction.
  * @throws When the AccountSet is Malformed.
  */
-// eslint-disable-next-line max-lines-per-function -- okay for this method, only a little over
+// eslint-disable-next-line max-lines-per-function, max-statements -- okay for this method, only a little over
 export function validateAccountSet(tx: Record<string, unknown>): void {
   validateBaseTransaction(tx)
 
+  if (tx.NFTokenMinter !== undefined && tx.NFTokenMinter === '') {
```

Comment:
I'd suggest comparing to https://github.com/XRPLF/xrpl.js/blob/main/packages/ripple-address-codec/src/xrp-codec.ts#L255 instead of checking for an empty string

`if (tx.NFTokenMinter !== undefined && !isValidClassicAddress(tx.NFTokenMinter)) {`

And then updating the tests for a few scenarios of bad addresses such as `null`, `''`, and `badaddress`

Reply:
I'm getting a type error when I'm using this function {} or null can't be passed as string

Reply:
@TusharPardhe that’s actually good as it should mean we don’t need tests since typescript is handling it

Reply:
I think it should still be tested.  Not everyone using the library uses typescript.

Reply:
Good point you can override with a ts-expect-error comment line

Reply:
The purpose of this method is to assist folks who aren't using typescript.

Reply:
I've added the additional check mentioned by @justinr1234